security/pfSense-pkg-suricata/files/usr/local/pkg/suricata/suricata_check_cron_misc.inc

```diff
@@ -382,37 +382,37 @@ if ($config['installedpackages']['suricata']['config'][0]['enable_log_mgmt'] ==
 
 		// Prune aged-out File Store captured files if any exist
 		if (is_dir("{$suricata_log_dir}/filestore") &&
-		    $config['installedpackages']['suricata']['config'][0]['file_store_retention'] > 0) {
+		    config_get_path('installedpackages/suricata/config/0/file_store_retention') > 0) {
 
-			$prune_count = suricata_check_prune_filestore_files("{$suricata_log_dir}/filestore", $config['installedpackages']['suricata']['config'][0]['file_store_retention']);
+			$prune_count = suricata_check_prune_filestore_files("{$suricata_log_dir}/filestore", config_get_path('installedpackages/suricata/config/0/file_store_retention'));
 			if ($prune_count > 0)
 				syslog(LOG_NOTICE, gettext("[Suricata] File Store captured files cleanup job removed {$prune_count} file(s) from '{$suricata_log_dir}/filestore/' path..."));
 		}
 
 		// Check File Store captured files storage limit and prune if necessary
 		if (is_dir("{$suricata_log_dir}/filestore") && 
-		    $config['installedpackages']['suricata']['config'][0]['file_store_limit_size'] > 0 && 
-		    (intval(suricata_folderSize("{$suricata_log_dir}/filestore")/1000000) >= $config['installedpackages']['suricata']['config'][0]['file_store_limit_size'])) {
-			suricata_check_filestore_limit_size("{$suricata_log_dir}/filestore", $config['installedpackages']['suricata']['config'][0]['file_store_limit_size']);
+		    config_get_path('installedpackages/suricata/config/0/file_store_limit_size') > 0 && 
+		    (intval(suricata_folderSize("{$suricata_log_dir}/filestore")/1000000) >= config_get_path('installedpackages/suricata/config/0/file_store_limit_size'))) {
+			suricata_check_filestore_limit_size("{$suricata_log_dir}/filestore", config_get_path('installedpackages/suricata/config/0/file_store_limit_size'));
 		}
 
 		// If a user-customized file store directory is set, check it, too
 		if (isset($value['file_store_logdir'])) {
 			if (is_dir($value['file_store_logdir']) && 
-			    $config['installedpackages']['suricata']['config'][0]['file_store_limit_size'] > 0 && 
-			    (intval(suricata_folderSize($value['file_store_logdir'])/1000000) >= $config['installedpackages']['suricata']['config'][0]['file_store_limit_size'])) {
-				suricata_check_filestore_limit_size($value['file_store_logdir'], $config['installedpackages']['suricata']['config'][0]['file_store_limit_size']);
+			    config_get_path('installedpackages/suricata/config/0/file_store_limit_size') > 0 && 
```

Comment:
Trailing whitespace

Reply:
Same as previous - removed the trailing space.